<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Replay1</class>
 <widget class="QWidget" name="Main_Replay1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Video_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>800</width>
     <height>440</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="Video_slider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Play_btn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.5x</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="curTime_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="totalTime_label">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>490</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
